### ISSUE TITLE
Improvement: for Antora, intellisense for anchors of included pages.

### ISSUE THREAD

**Issue by danhaywood** [asciidoctor/asciidoctor-intellij-plugin#403]
I use include::'s to break up a large page into smaller onces, and use leveloffset=+1 so that the H1 of the included pages becomes an H2 (with implicit anchor) in the parent page that includes them.

so given:
* parent.adoc
* parent/child-1.adoc
* parent/child-2.adoc

in parent.adoc I have:

    include::parent/child-1.adoc[leveloffset=+1]
    include::parent/child-2.adoc[leveloffset=+1]

Where in child-1.adoc it is:

    = Child-1
    
    yada yada

I'd like Intellisense to be able to find the implicit "child-1" anchor.

In other words, when I type:

xref:parent.adoc#

and hit ctrl-enter, then I'd like "child-1", "child-2" to be offered as anchors.

**Comment by ahus1** [comment]
Thanks for suggesting this, this will be part of the next pre-release. 
When trying the Apache ISIS docs, I found some include missing a colon; see PR https://github.com/apache/isis/pull/174 for some fixes.

I'll update this issue again once a pre-release is available.

**Comment by ahus1** [comment]
@danhaywood - the key for auto-complete for anchors on windows would be Ctrl+Space. This is what I tested and what worked for me.

**Comment by danhaywood** [comment]
Ok, thanks, I'll try that out.

**Comment by ahus1** [comment]
Pre-release 0.30.38 is available. Please give it a try and let me know if this works for you.

The new pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/CONTRIBUTING.adoc#eap).
